crates/bevy_color/src/color_ops.rs

```diff
@@ -97,6 +99,26 @@ pub trait ClampColor: Sized {
     fn is_within_bounds(&self) -> bool;
 }
 
+/// Trait with methods for converting colors to non-color types
+pub trait Convert {
```

Comment:
I think we could do better on the trait naming. Some suggestions:
* RawColorConversion
* ColorLowering - "lowering" is a term used in compilers to mean transforming to a lower-level representation.
* ColorVecConversions
* ColorToComponents
* ColorToArray

Reply:
+1 for `ColorToComponents`. Descriptive without tying itself to any particular type.

Reply:
ColorToRaw? throwing another one out but these are all good!